modules/osdk-scorecard-olm.adoc

```diff
@@ -0,0 +1,217 @@
+                ...
+----
+<1> Scorecard `kubeconfig` volume mount.
+<2> Scorecard `kubeconfig` environment variable.
+<3> Repeat the same for this and all other containers.
+
+. Insert the scorecard proxy container into the Operator's Deployment:
++
+[source,yaml]
+----
+spec:
+  install:
+    spec:
+      deployments:
+      - name: memcached-operator
+        spec:
+          ...
+          template:
+            ...
+            spec:
+              containers:
+              ...
+              - name: scorecard-proxy <1>
+                command:
+                - scorecard-proxy
+                env:
+                - name: WATCH_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      apiVersion: v1
+                      fieldPath: metadata.namespace
+                image: quay.io/operator-framework/scorecard-proxy:master
+                imagePullPolicy: Always
+                ports:
+                - name: proxy
+                  containerPort: 8889
+----
+<1> Scorecard proxy container.
+
+[TIP]
+====
+Alternatively, the
+link:https://github.com/operator-framework/community-operators[`community-operators`]
+repository has several bash functions that can perform this procedure for you:
+
+----
+$ curl -Lo csv-manifest-modifiers.sh \
+    https://raw.githubusercontent.com/operator-framework/community-operators/master/scripts/lib/file
+$ . ./csv-manifest-modifiers.sh
+$ create_kubeconfig_secret_file scorecard-secret.yaml "<namespace>" <1>
+$ oc apply -f scorecard-secret.yaml
+$ insert_kubeconfig_volume "<csv_file>" <2>
+$ insert_kubeconfig_secret_mount "<csv_file>"
+$ insert_proxy_container "<csv_file>" "quay.io/operator-framework/scorecard-proxy:master"
+----
+<1> Replace `<namespace>` with the namespace your Operator will deploy in.
+<2> Replace `<csv_file>` with the path to your Operator's CSV manifest.
+====
+
+After completing this procedure, follow the steps in
+link:https://github.com/operator-framework/community-operators/blob/master/docs/testing-operators.md[Testing your Operator with Operator Framework]
+to bundle your CSV and CRDs, deploy OLM on minikube or
+link:https://www.okd.io/[OKD], and deploy your Operator.
+
+After your Operator has been deployed, run the scorecard with both the
+`csv-path: <csv_manifest_path>` and `olm-deployed` options set.
+
+[NOTE]
+====
+Currently, using the scorecard with a CSV does not permit multiple CR manifests
+to be set through the CLI/config/CSV annotations. You must tear down your
+Operator in the cluster, re-deploy, and re-run the scorecard for each CR being
+tested. In the future, the scorecard will fully support testing multiple CRs
+without requiring users to teardown and standup each time.
+====
+
+[NOTE]
+====
+You can either set `cr-manifest` or your CSV's
+link:https://github.com/operator-framework/operator-lifecycle-manager/blob/master/doc/design/building-your-csv.md#crd-templates[`metadata.annotations['alm-examples']`]
```

Comment:
I think this is where your build issue is coming from

```
Opening and ending tag mismatch: literal line 7604 and link, line 7604, column 196 (master.xml, line 7604)
```

Reply:
Yay thanks.